protocol/contracts/reserve/Cargo.toml

```
@@ -47,6 +55,16 @@ combinations = [
         "net_main",
         "neutron-astroport-usdc_noble",
     ], include-rest = false },
+    { tags = [
+        "ci",
+        "net_main",
+        "osmosis-osmosis-statom",
+    ], feature-groups = [
+        "testing",
```

The above lines are missing `"contract-or-stub"` too.

```suggestion
        "contract-or-stub",
        "testing",
```